### ISSUE TITLE
Simplewallet infinite loop on transfer

### ISSUE THREAD

**Issue by pmitchev** [amjuarez/bytecoin#41]
I try to send a large amount with simplewallet and the transfer command seems to enter it in an infinite loop. Here is what i am trying:
```
transfer 0 D4yvJXGDfRbQ2XFaji3WeNA8wEjf1D279MMUkcCzx2xjabXD6QR9DiWDgLMPp48krGUS6JErJZeeWEGp3wT61d7iCS6hfTp 112351254
```
The result is wallet stuck, and it has to be force killed.

**Comment by amjuarez** [comment]
Hello,

1) Run simplewallet, make it stuck.
2) Run following commands

$ ps aux | grep simplewallet
user     29350  0.0  0.0 197964  3484 pts/25   Sl+  12:57   0:00 ./src/simplewallet

3) Use pid 29350 obtained from above:

$ sudo gdb -p 29350

4) Run gdb command:

(gdb) thread apply all bt

5) Send us the result of this command. 

Thank you.

**Comment by pmitchev** [comment]
https://gist.github.com/pmitchev/10057814d8ee75576679

**Comment by amjuarez** [comment]
Thank you for reporting. We are currently investigating it and will update you soon.

**Comment by amjuarez** [comment]
Hello, 
This issue has been fixed in the 1.0.4 release. Please let us know if problem still present.

**Comment by pmitchev** [comment]
Hello,
It looks fixed. Now it returns "Error: Transaction size is too big", which is the expected behaviour.
